### PR TITLE
Removed links to the mobile apps (iPhone/Android).

### PR DESCRIPTION
**Pull request by cedricbonhomme**
Just to remove some dead links.

### CODE REVIEW

File: src/html/assets/themes/fabrizio1.0/templates/partials/header-secondary.php [hunk 1]
```diff
@@ -17,10 +17,6 @@
                 <li><a href="#" class="triggerShare"><i class="icon-share-alt triggerShare"></i> Share this album</a></li>
               <?php } ?>
             <?php } ?>
-          <?php } elseif($this->utility->isActiveTab('upload')) {?>
```
Comment (jmathai) [inline src/html/assets/themes/fabrizio1.0/templates/partials/header-secondary.php]: We should be able to keep these links in place. The official Trovebox app was removed but there is [another iOS app](https://github.com/delightfulapp/delightful) by @delightfulapp. The iTunes link is here, https://itunes.apple.com/us/app/delightful-trovebox-photo/id878230610?mt=8

Is there an APK we can upload somewhere so folks can install the Android build? @patricksan

Comment (cedricbonhomme) [inline src/html/assets/themes/fabrizio1.0/templates/partials/header-secondary.php]: Yes, if we have applications available on the store., it makes sense to keep theses links.

**Comment by cedricbonhomme** [comment]
I close this PR since there is an app for iOS. It is an "official" app?
We can see later for Android. My personal instance of Trovebox is already running without theses links (on a dedicated branch).

**Comment by patricksan** [comment]
@jmathai and @cedricbonhomme for the Android app we could upload the APK to Google Play. I just need to check if the binaries are ok. Also, I would like to remove the old login we have it. Added to my todo list

**Pull request closed without merging**